.github/workflows/build_wamr_vscode_ext.yml

```diff
@@ -20,10 +20,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Use Node.js 14.x
```

Comment:
What was the reason for specifying a particular node version here in the first place? @wenyongh

Reply:
I am not sure, the code was written by me, but I think it is to specify a runnable version here. Upgrading to 16.x should be OK since14.x is a little old.